lib/build.js

```diff
@@ -906,10 +906,10 @@ const buildAct = config => ({act, oog, pass, name, hasGas, gas, lemma}) => {
       && ("_ => " + buildReturns(act.returns))
       || act.returnsRaw && "_ => " + act.returnsRaw
       || ".WordStack"))
-    || "_ => _";
+    || "_ => ?_";
```

Comment:
@MrChico should this (and all other occurrences of `_ => _` in klab just be `_`?